extensions.json

```
@@ -105,8 +105,8 @@
     {
       "id": "arcanis.vscode-zipfs",
       "repository": "https://github.com/yarnpkg/berry",
-      "version": "2.2.4",
-      "checkout": "vscode-zipfs/2.2.4",
+      "version": "2.3.0-rc.2",
+      "checkout": "vscode-zipfs/2.3.0-rc.2",
```

We shouldn't auto-upgrade to `-rc` versions:

```suggestion
      "version": "2.2.4",
      "checkout": "vscode-zipfs/2.2.4",
```